### PR TITLE
Pull request from Dario's branch

### PR DESCRIPTION
**Pull request by darko-h4**
I refactored the code a bit and add a brief description of the project on the README file.

### CODE REVIEW

**Comment by mircealungu** [comment]
Ehi folks, what group are you? :)

**Comment by darko-h4** [comment]
We are Group 10! Sorry, I specified it in the description of the README
file in the pull request.

Il mar 20 ott 2020, 22:29 Mircea Filip Lungu <notifications@github.com> ha
scritto:

**Comment by mircealungu** [comment]
I saw it after I wrote. Thanks for the prompt reply!

**Comment by mircealungu** [comment]
Also, if this is your submission, don't forget to tag the staff so they get notified!

File: src/homepages.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+export const firstHomepage = {
```
Comment (mircealungu) [inline src/homepages.js:1]: by definition there is only one "homepage" for a website.
are you sure that you have three "homepages"?
if not, maybe rename to something that's more "intention revealing".

**Comment by darko-h4** [comment]
@madsnedergaard

**Comment by madsnedergaard** [comment]
> @madsnedergaard

Thanks - in future you can use the "assign a reviewer" feature here:

image

**Comment by madsnedergaard** [comment]
Something is wrong with your README 🤔 

Screenshot 2020-10-20 at 22 38 05

File: src/Homepage.js [hunk 2]
```diff
@@ -1,5 +1,9 @@
 import React, {Component} from 'react';
 
+const SwipePage = () => {
+    //Swipe
+}
+
 class Homepage extends Component { 
 
     constructor(props)
```
Comment (madsnedergaard) [inline src/Homepage.js]: Is there a reason why you're using classes? If you saw the first lecture about React you know I really don't like them in new code 😉

**Review by madsnedergaard — COMMENTED** [review]
Going forward, please try to only work on `development` and not `main` branch. And then when you need to do these "hand-ins", you create a PR for merging `development` into `main`. That way we can see what you did and how you did it, and we can actually comment on the code :)

I'd suggest that you add a branch protection rule like this, so you don't accidentally commit things to the wrong branch:

Screenshot 2020-10-21 at 13 05 11

image

File: src/Homepage.js [hunk 3]
```diff
@@ -1,28 +1,21 @@
-import React, {Component} from 'react';
```
Comment (madsnedergaard) [inline src/Homepage.js:1]: Thanks for changing this! Just FYI, this is now a "functional component", whereas "hooks" is some built-in utility functions we can use to enhance our components (and use state, etc. in them)

**Review by Akons97 — COMMENTED** [review]
Have you worked on this since the last week? I cannot see a new PR
What is your development branch? I can see that all of you have your own

**Comment by darko-h4** [comment]
I didn't yet, still trying to understand with he group what to do. Probably
I'll be working on it this weekend

Il gio 5 nov 2020, 13:15 Adam <notifications@github.com> ha scritto:

**Pull request closed without merging**